src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -240,10 +247,12 @@ WorkspaceInitialPage.displayName = 'WorkspaceInitialPage';
 
 export default compose(
     withLocalize,
-    withFullPolicy,
```

Comment:
Sorry for the confusion, but we're no longer looking to remove `withFullPolicy` completely. Instead, we'll just be updating it to not fetch policy data so you can keep it here

Reply:
Hmmm... Yes, I am confused 😅 . Do we _need_ `withFullPolicy` here - why keep it? Also, the `withFullPolicy` code is still loading the full policy - when are we removing that?

Reply:
So `withFullPolicy` also tracks the last visited workspace/policy so we shouldn't remove that functionality.

We'll remove the policy loading with [this GH](https://github.com/Expensify/Expensify/issues/217650) but this is only safe to do once all the other refactors are done (if we do it before, these pages will have no data).

Reply:
Okay great - makes sense thanks for clarifying!